core/playwright/pub.spec.ts

```diff
@@ -196,10 +203,11 @@ test.describe("Creating a pub", () => {
 		await page.getByLabel("Animals").fill("penguins");
 		await page.keyboard.press("Enter");
 		await page.getByTestId("remove-button").first().click();
+		await page.waitForTimeout(200);
```

Comment:
this was erroring without this timeout for some reason locally, grrrr

Reply:
ez fix. make every request blocking and disable all animations